org.eclipse.cdt.cross.arc.gnu/plugin.xml

```diff
@@ -679,7 +679,72 @@
             name="Use memory map from TCF"
             valueType="boolean">
       </option>
-
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            defaultValue="false"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.target.includecdefines"
+            name="Include C defines from TCF"
+            valueType="boolean">
+      </option>
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.specsem"
+            name="Use GCC specs file (ARC EM)"
+            valueType="enumerated">
+         <enumeratedOptionValue
+               command="--specs=nsim.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nsim"
+               isDefault="true"
+               name="nsim">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               command="--specs=emsk_em9d.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.emsk_em9d"
+               name="emsk_em9d">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               command="--specs=emsk_em11d.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.emsk_em11d"
+               name="emsk_em11d">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               command="--specs=nosys.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nosys"
+               name="nosys">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.none"
+               name="None">
+         </enumeratedOptionValue>
+      </option>
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.specs"
+            name="Use GCC specs file"
+            valueType="enumerated">
+         <enumeratedOptionValue
+               command="--specs=nsim.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nsim"
+               isDefault="true"
+               name="nsim">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               command="--specs=nosys.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nosys"
+               name="nosys">
+         </enumeratedOptionValue>
+         <enumeratedOptionValue
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.none"
+               name="None">
+         </enumeratedOptionValue>
+      </option>
+      <option
+            command="--specs=nano.specs"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nano"
+            name="Use nano.specs"
```

Comment:
Also since another change is needed anyway, please improve this text: **Enable nano.specs to use standard libraries optimized for code size**.